docs/fides/docs/tutorial/annotate.md

```diff
@@ -0,0 +1,266 @@
+    - name: password
+      data_categories: [user.provided.identifiable.credentials.password]
+      data_qualifier: aggregated.anonymized.unlinked_pseudonymized.pseudonymized
+```
+
+<!-- TODO: Link to the `generate-dataset` usage documentation below, when it exists. -->
+
+As an alternative to manually authoring the resource file, you can also use the `generate-dataset` CLI command. The CLI will connect to the database and automatically generate a non-annotated resource YAML file in the specified location, based on the database schema. For this project, the command is:
+
+```sh
+./venv/bin/fidesctl generate-dataset \
+postgresql://postgres:postgres@localhost:5432/flaskr \
+fides_resources/flaskr_postgres_dataset.yml
+```
+
+### Understanding the Dataset Resource
+
+This YAML serves as the foundation of `fideslang`, the Fides language; it answers "_What data and kinds of data do we have?_" and "_How is our data organized?_". The language is built on declaring the types of data found in storage for your organization.
+
+In traditional SQL, Fidesctl defines the following:
+- "datasets" as database schemas
+- "collections" as database tables
+- "fields" as database columns
+
+For NoSQL datasets, Fidesctl defines the following:
+- "dataset"
+- "collection" as a logical grouping of data fields (ie: in MongoDB, this is called a "Collection")
+- "fields" as a reference to an individual data element (ie: in MongoDB, this is called a "field")
+
+Additionally, `fideslang` has attributes that describe what kind of data is contained in this dataset. We use the following attributes to describe the data:
+
+| Name | Type | Description |
+| --- | --- | --- |
+| name | String | The name of this field |
+| description | String | A description of what this field contains |
+| data_categories | List[FidesKey] | The types of sensitive data, as defined by the taxonomy, that can be found in this field |
+| data_qualifier | FidesKey | The level of deidentification for the dataset |
+
+---
+
+**PRO TIP**
+
+As you're progressing with the tutorial, we recommend installing our [FidesCTL VS Code extension](https://marketplace.visualstudio.com/items?itemName=ethyca.fidesctl), which will validate the syntax in real-time as you're writing your resource files!
+
+---
+
+### Maintaining a Dataset Resource
+
+As apps add more databases and other services to store potentially sensitive data, it is recommended that updating this resource file becomes a part of the development process when building a new feature.
+
+## Annotate the System
+
+Now that you've built out the underlying database that describes how and what type of data is stored, include the database in application-level "systems", another critical Fidesctl resource.
+
+This app contains a single Flaskr Web Application system resource. Create a `system` resource to annotate it by adding a `flaskr_system.yml` file to the `fides_resources` directory. This file should contain the following configuration:
```

Comment:
This needs a quick description about how one should consider annotating a system. In this case, something like:
> Writing a Fides privacy declaration for our system involves stepping back and answering the questions: what data is this system processing? Why is it processing it? Whose data is involved? How is it protected?

> Fides answers these questions with a `privacy_declaration` that describes the data categories, use, subjects, and qualifier. Our example application is quite simple so it only has a single use: to _provide_ the service to users. To provide the service, it needs some identifiable data (like name, email, contact, etc.) and it also derives some data too (like unique IDs).

...etc.

I think it'll just help humanize the declarations by giving an English paragraph description about why these annotations make sense